### PR TITLE
solve issue and improve rendering

### LINKED ISSUE #37: Add copyright in documentation footer
Add " (c) 2024 RTE France. Unless otherwise indicated, content is licensed under CC-BY-4.0." in the footer

Optional : add the following links
  links:
    - icon: "fab fa-fw fa-square-x-twitter"
      url: https://twitter.com/rte_france
    - icon: "fab fa-fw fa-youtube-square"
      url: https://www.youtube.com/user/myRTEtube
    - icon: "fab fa-fw fa-linkedin"
      url: https://www.linkedin.com/company/rte_245681
    - icon: "fab fa-fw fa-square-instagram"
      url: https://www.instagram.com/rte_france

### PR DESCRIPTION
**Pull request by adriengoeller**
**Please check if the PR fulfills these requirements**

- [ X] Tests for the changes have been added (for bug fixes / features)
- [ X] Docs have been added / updated (for bug fixes / features)

**Does this PR already have an issue describing the problem?**

Fixes #37

**What kind of change does this PR introduce?**

Display documentation issue with repeating first header in the auto documentation from docstring

**What is the current behavior?**

**What is the new behavior (if this is a feature change)?**

**Does this PR introduce a breaking change or deprecate an API?**
- [ ] Yes
- [ X] No

**If yes, please check if the following requirements are fulfilled**

- [ ] The *breaking change* or *deprecated* label has been added
- [ ] The migration steps are described in the following section

**What changes might users need to make in their application due to this PR? (migration steps)**

**Other information**:

### CODE REVIEW

File: docs/docstring/welcome.md
Comment (ai-qui) [inline docs/docstring/welcome.md:1]: There's a merge conflict on this file, that must be resolved before this PR can be merged.
Also, I noticed that in the updated User Guide, the documentation is missing for the classes `CatenaryCableModel` and  GeometricCableModel`.

I guess that's because the work on this PR was started before these two classes were merged into `main`.

The following steps should fix this:
1. merging `main` into this branch
2. adding a markdown configuration file (like `docs/docstring/entities/schemas.md` ) for the module `mechaphlowers.core.models.cable_models` which defines these classes.

File: mkdocs.yaml [hunk 1]
```diff
@@ -54,22 +64,62 @@ plugins:
...
   - javascripts/mathjax.js
   - https://unpkg.com/mathjax@3/es5/tex-mml-chtml.js
 
-# extra:
+extra:
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/phlowers
+    - icon: fontawesome/brands/twitter
+      link: https://twitter.com/rte_france
```
Comment (ai-qui) [inline mkdocs.yaml:117]: I wonder whether we should use the new URL and logo for X?

File: mkdocs.yaml [hunk 2]
```diff
@@ -54,22 +64,62 @@ plugins:
...
+
 extra_javascript:
   - javascripts/mathjax.js
   - https://unpkg.com/mathjax@3/es5/tex-mml-chtml.js
 
-# extra:
+extra:
+  social:
+    - icon: fontawesome/brands/github
+      link: https://github.com/phlowers
```
Comment (ai-qui) [inline mkdocs.yaml:115]: I suggest adding a `name` attribute to each of these links, in order to accessibility (for instance for people using screenreaders). For example:

```suggestion
    - icon: fontawesome/brands/github
      link: https://github.com/phlowers
      name: phlowers on Github
```

From [mkdocs documentation](https://squidfunk.github.io/mkdocs-material/setup/setting-up-the-footer/#+social.name):
> name:
> domain name from link, if available This property is used as the link's title attribute and can be set to a discernable 
> name to improve accessibility

**Review by ai-qui — CHANGES_REQUESTED** [review]
Nice work with the headers, footers and title hierarchy!
Also I like the new "search highlight" feature and the colors in the updated User Guide.

I left some comments, I let you review them.

**Review by yousza — APPROVED** [review]
